components/member-profile/contribution/index.js

```diff
@@ -10,11 +11,70 @@ const renderPRLinks = (prList) =>
 
 const Contribution = ({ contribution, fullName, imageLink, devUser }) => {
   const {
-    task: { title, startedOn, endsOn, status, purpose, featureUrl },
+    task: { featureUrl },
+    prList
+  } = contribution;
+  const url = featureUrl || prList[0]['url'];
+  const gotoUrl = () => url && window.open(url, '_blank');
+  const urlObj = url && new URL(url);
+  const card = () => (
+    <ContributionCard
+      contribution={contribution}
+      fullName={fullName}
+      imageLink={imageLink}
+      devUser={devUser}
+      url={url}
+      urlObj={urlObj}
+    />
+  );
+  const renderFeatureCard = () => {
+    if (urlObj?.host === 'members.realdevsquad.com') {
+      return (
+        <Link href={urlObj.pathname}>
+          <div className={url && classNames.contributionCard}>{card()}</div>
+        </Link>
+      );
+    } else {
+      return (
+        <div
+          className={url && classNames.contributionCard}
+          onClick={gotoUrl}
+          onKeyDown={gotoUrl}
+          aria-hidden="true">
+          {card()}
+        </div>
+      );
+    }
+  };
+
+  return renderFeatureCard();
+};
+
+const ContributionCard = ({ contribution, fullName, imageLink, devUser, url, urlObj }) => {
+  const {
+    task: { title, startedOn, endsOn, status, purpose },
     prList
   } = contribution;
   const isTitleAvailable = title ? true : false;
   const featureTitle = isTitleAvailable ? title : prList[0].title;
+  const renderFeatureUrl = (url, urlObj) => {
+    if (url) {
```

Comment:
I got to learn a lot from this PR, I had a question here , instead of using nested if's and passing return;
Could we go for conditional rendering here?
Like {URL &&(what we need to render here)}

What would be better practise here @ankushdharkar

Reply:
DO we really need to check this condition? I think the `url` will always be true? 🤔

Reply:
@sumitd94 ankush gave me a case if there is no URL then what